docs/querying/lookups-cached-global.md

```diff
@@ -22,21 +26,22 @@ title: "Globally Cached Lookups"
   ~ under the License.
   -->
 
+Globally cached lookups all draw from the same cache pool, allowing each Druid process to have a fixed cache pool that can be used by cached lookups.
+
 To use this Apache Druid extension, [include](../configuration/extensions.md#loading-extensions) `druid-lookups-cached-global` in the extensions load list.
 
 ## Configuration
-:::info
- Static configuration is no longer supported. Lookups can be configured through
- [dynamic configuration](./lookups.md#configuration).
-:::
 
 Globally cached lookups are appropriate for lookups which are not possible to pass at query time due to their size,
 or are not desired to be passed at query time because the data is to reside in and be handled by the Druid servers,
 and are small enough to reasonably populate in-memory. This usually means tens to tens of thousands of entries per lookup.
 
-Globally cached lookups all draw from the same cache pool, allowing each process to have a fixed cache pool that can be used by cached lookups.
+:::info
+ Druid no longer supports static lookup configuration. You can configure lookups through
+ [dynamic configuration](./lookups.md#configuration).
+:::
```

Comment:
how long has this been out of support? Can we just not mention now?

Reply:
yes, updated